RoadRunner to unreal read the docs addition (#318)
ni/monodrive-documentation: docs/unreal_tutorials/RoadRunnertoUnrealpt1.md

# monoDrive: RoadRunner to Unreal Pipeline 1

### Overview

These walkthroughs go through each of the monoDrive methods of bringing the maps created in RoadRunner into Unreal and to get the
 For more information, see [monodrive Documentation](https://monodrive.readthedocs.io/en/latest/).

For [Unreal Engine Tutorials](https://monodrive.readthedocs.io/en/latest/unreal_tutorials/MayaWheels/) for guides.

For [Maya Tutorial](https://knowledge.autodesk.com/search-result/caas/simplecontent/content/maya-1-minute-startup-movies.html) for guides.

To continue with the Unreal importation continue to the RoadRunnertoUnreal Part 2.

#### Roadrunner Basics

Overview

This section of the walkthrough goes through the meathods of creating basic roads in RoadRunner as well as how to export them in preperation for Unreal Engine 4.

NOTE: RoadRunner uses Meters as its base measurement scale. Importing a vehicle will help you with the scale of your scene.

#### Road Creation Basics

- Open RoadRunner
    - Create a new scene/map:
  <div class="img_container">
    <img class='lg_img' src="../imgs/RoadRunnerPic_01.png"/>
  </div>

> - With the 'Road Plan Tool' selected, located in the top left of the screen:

Reviewer: ```suggestion
1. Select the 'Road Plan Tool' icon located in the top left
```

<div class="img_container">

>     <img class='sm_img' src="../imgs/RoadRunnerPic_02.png"/>

Reviewer: The medium image is easier to view.

```suggestion
    <img class='md_img' src="../imgs/RoadRunnerPic_02.png"/>
```

</div>

> - Navigate to the 'Library Browser' in the bottom right of the screen

Reviewer: ```suggestion
1. Navigate to the 'Library Browser' in the bottom right of the screen
```

    - Navigate to the folder labeled 'Road Styles'

>     - Select the road type you would like to use

Reviewer: ```suggestion
    - Select road type
```

<div class="img_container">

>     <img class='lg_img' src="../imgs/RoadRunnerPic_03.png"/>

Reviewer: I can not read any of the text in this picture, maybe retake?

</div>

> - Now with the 'Road Plan Tool' selected

Reviewer: delete this line since it's already a step

>   1. Right click anywhere on the grid that you see, this will create a red dot

Reviewer: ```suggestion
1. Right click on the grid, this will create a red dot
```

>   2. Next 'right' click again in another location, this will create your basic road.

Reviewer: I added some clarification here, please make sure that I clarified correctly. 
```suggestion
1. When right clicking again in another location within the grid, this will create the basic road selected.
```
